test/python/tools/test_country_info.py

```diff
@@ -59,3 +59,39 @@ def test_create_country_names(temp_db_with_extensions, temp_db_conn, temp_db_cur
     else:
         assert result_set == {'us' : set(('us', 'us1', 'us2', 'United States')),
                               'fr' : set(('fr', 'Fra', 'Fren'))}
+
+
+def test_setup_country_config_languages_not_loaded(project_env, def_config):
+    (project_env.project_dir / 'country_settings.yaml').write_text("""
+de:
+    partition: 3
+    names: 
+        name: 
+            default: Deutschland
+""")
+    country_info.setup_country_config(def_config)
```

Comment:
Ah, sorry. `project_env` is already the configuration (see `test/python/conftest/`), so you would want to use `country_info.setup_country_config(project_env)`